src/applications/static-pages/facilities/tests/NearbyLocations.unit.spec.jsx

```diff
@@ -0,0 +1,528 @@
+    type: 'facility',
+    attributes: {
+      access: {
+        health: [],
+        effectiveDate: null,
+      },
+      activeStatus: 'A',
+      address: {
+        mailing: {},
+        physical: {
+          zip: '44106',
+          city: 'Cleveland',
+          state: 'OH',
+          address1: '8901 Superior Avenue',
+          address2: null,
+          address3: null,
+        },
+      },
+      classification: 'Other Outpatient Services (OOS)',
+      detailedServices: [],
+      facilityType: 'va_health_facility',
+      feedback: {
+        health: {},
+        effectiveDate: null,
+      },
+      hours: {
+        friday: '730AM-500PM',
+        monday: '730AM-500PM',
+        sunday: 'Closed',
+        tuesday: '730AM-500PM',
+        saturday: 'Closed',
+        thursday: '730AM-500PM',
+        wednesday: '730AM-500PM',
+      },
+      id: 'vha_541GM',
+      lat: 41.52075514,
+      long: -81.62566982,
+      mobile: false,
+      name: 'Cleveland VA Clinic-Superior',
+      operatingStatus: {
+        code: 'NORMAL',
+      },
+      operationalHoursSpecialInstructions: null,
+      phone: {
+        fax: '216-421-3134',
+        main: '216-421-3133',
+        pharmacy: '216-791-3800 x62001',
+        afterHours: '216-791-3800 x64441',
+        patientAdvocate: '216-791-3800 x61700',
+        enrollmentCoordinator: '216-791-3800 x65189',
+        healthConnect: null,
+      },
+      services: {
+        other: [],
+        health: [],
+        lastUpdated: null,
+      },
+      uniqueId: '541GM',
+      visn: '10',
+      website:
+        'https://www.va.gov/northeast-ohio-health-care/locations/cleveland-va-clinic-superior/',
+    },
+  },
+];
+
+const mainVetCenterAddress = {
+  countryCode: 'US',
+  administrativeArea: 'MI',
+  locality: 'Escanaba',
+  postalCode: '49829',
+  addressLine1: '3500 Ludington Street',
+  addressLine2: 'Suite 110',
+  organization: null,
+};
+
+const mainVALocationAddress = {
+  countryCode: 'US',
+  administrativeArea: 'OH',
+  locality: 'Cleveland',
+  postalCode: '44199',
+  addressLine1: '1240 E. 9th Street, A.J. Celebrezze Federal Building',
+  addressLine2: null,
+  organization: null,
+};
+
+describe('NearbyVetCenters', () => {
+  it('should render spinner while loading', () => {
+    const state = {
+      facility: { loading: true },
+    };
+    const fakeStore = createFakeStore(state);
+    const wrapper = mount(
+      <Provider store={fakeStore}>
+        <NearbyVetCenters />
+      </Provider>,
+    );
+    expect(wrapper.find('va-loading-indicator').exists()).to.be.true;
+    wrapper.unmount();
+  });
+
+  it('should not render spinner if not loading', () => {
+    const state = {
+      facility: { loading: false },
+    };
+    const fakeStore = createFakeStore(state);
+    const wrapper = mount(
+      <Provider store={fakeStore}>
+        <NearbyVetCenters />
+      </Provider>,
+    );
+    expect(wrapper.find('va-loading-indicator').exists()).to.be.false;
```

Comment:
**Suggestion:** Should this test also check to ensure that a part or wrapper for the component itself also loads? If the loader isn't there, but the component itself fails, this test could have a misleading positive result.

Reply:
Hmm, let me think about what does actually load in the scenario.

Reply:
wrapper seems to crash if there is a problem loading the component (I added a thrown error) and as a result expect fails (is NOT false)